src/app/showcase/colors-showcase/colors-showcase.component.scss

```
@@ -3,62 +3,114 @@
 @each $color, $color-name in $kirby-colors {
   .#{$color} {
     background-color: var(--kirby-#{$color});
```

Consider using `get-color` function as we try to promote that everywhere:
```suggestion
    background-color: #{get-color($color)};
```